### PR TITLE
CSS 15.04.2021

### PR DESCRIPTION
**Pull request by tomriiich**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,38 +1,101 @@
...
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  background: linear-gradient(#202020, #383838);
+  /* background-repeat: no-repeat;
+  background-image: url("https://nsa40.casimages.com/img/2021/04/15/21041509430970619.jpg");
+  background-color: whitesmoke;
+  background-position: center top; */
```
Comment (rocambille) [inline src/App.css]: remove unused code :)

File: src/components/Card.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+}
+
+.heroName {
+    font-size: 30px;
+    margin-bottom: 15px;
+    color: #2dffe0;
+}
+
+.cardText {
+    /* line-height: 0px; */
```
Comment (rocambille) [inline src/components/Card.css]: remove unused code :)

File: src/App.css [hunk 3]
```diff
@@ -1,38 +1,101 @@
-.App {
+@import url('https://fonts.googleapis.com/css2?family=Oswald:wght@200;300;400;500;600;700&display=swap');
+
+* {
+    margin: 0;
+    padding: 0;
+    font-family: oswald;
+    /*min-height: 100vh;*/
```
Comment (rocambille) [inline src/App.css]: remove unused code :)

**Review by rocambille — CHANGES_REQUESTED** [review]

File: src/App.css (hunk 1 above)
Comment (tomriiich) [reply src/App.css]: Done

**Pull request merged**